src/main/java/fr/darkvodou/NKtrader/managers/TraderManager.java

```diff
@@ -0,0 +1,59 @@
+package fr.darkvodou.NKtrader.managers;
+
+import fr.darkvodou.NKtrader.entity.Trader;
+import org.bukkit.command.ConsoleCommandSender;
+
+import java.util.HashMap;
+
+import static fr.darkvodou.NKtrader.enums.Msgmanager.*;
+
+public class TraderManager
+{
+	private HashMap<String, Trader> traders = new HashMap<>();
+
+	public ConsoleCommandSender consoleCommandSender;
+
+	public TraderManager(ConsoleCommandSender consoleCommandSender)
+	{
+		this.consoleCommandSender = consoleCommandSender;
+	}
+
+	public boolean hasTrader(String name)
+	{
+		return traders.containsKey(name);
+	}
+
+	public void addTrader(String name, Trader trader)
+	{
+		if(hasTrader(name))
+			consoleCommandSender.sendMessage("" + ERROR_TRADER_IS_CONTAINED);
```

Comment:
use { }
sendMessage(ERROR_TRADER_IS_CONTAINED.toString());
add "return;" and remove the else